components/upload/events.md

```diff
@@ -179,6 +181,13 @@ public async Task<IActionResult> Save(IFormFile files)
 
 See the [full example](#example) below.
 
+## OnPause
```

Comment:
All the other events have minor skipped from REPL examples. I suggest adding such for OnPause and OnResume for consistency. Otherwise, we have to refactor the whole article to remove the other sample examples for each event.

Reply:
I also recommend adding the arguments in a table - applies to both new events.